app/controllers/jobs_controller.rb

```diff
@@ -1,8 +1,39 @@
+require 'ostruct'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.